src/tasks/DockerPseudoterminal.ts

```diff
@@ -3,10 +3,13 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as nodeptytypes from 'node-pty';

```

Comment:
Build is failing because I don't have `node-pty.d.ts` available. Need to find a way to download it to `node_modules`.